code/src/components/Emoji.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const Emoji = props => (
+    <span
+        className="emoji"
+        role="img"
+        aria-label={props.label ? props.label : ""}
+        aria-hidden={props.label ? "false" : "true"}
```

Comment:
Good with using accessibility here!! Looks good with the ternary oparator as well